examples/metrics-basic/src/main.rs

```diff
@@ -1,12 +1,17 @@
 use opentelemetry::global;
 use opentelemetry::KeyValue;
-use opentelemetry_sdk::metrics::{PeriodicReader, SdkMeterProvider};
+use opentelemetry_sdk::metrics::{data::Temporality, PeriodicReader, SdkMeterProvider};
 use opentelemetry_sdk::{runtime, Resource};
 use std::error::Error;
 use std::vec;
 
 fn init_meter_provider() -> opentelemetry_sdk::metrics::SdkMeterProvider {
-    let exporter = opentelemetry_stdout::MetricsExporterBuilder::default().build();
+    let exporter = opentelemetry_stdout::MetricsExporterBuilder::default()
+        // Build exporter using Default (Cumulative) Temporality.
+        .with_temporality(Temporality::default())
```

Comment:
For a better user experience, let's not put any examples that explicitly use `Temporality::default()`. It should be enough to have a comment mention that : by default, `Cumulative` temporality will be used.

We don't want to give the impression that one needs to explicitly add `.with_temporality(Temporality::default())` to use `Cumulative` aggregation.